### PR TITLE
store/tikv: add log.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM
@tiancaiamao PTAL

File: store/tikv/region_cache.go [hunk 1]
```diff
@@ -416,6 +416,8 @@ func (c *RegionCache) OnRequestFail(ctx *RPCContext) {
 	delete(c.storeMu.stores, storeID)
 	c.storeMu.Unlock()
 
+	log.Warnf("drop regions from cache due to request fail, storeID: %v", storeID)
```
Comment (tiancaiamao) [inline store/tikv/region_cache.go]: Is it better to log this as info? This case is normal on bad network, leader change, region split, and so on....

Comment (disksing) [inline store/tikv/region_cache.go]: For leader change or region split, server will response a NotLeader or RegionNotFound. So just network timeout will cause request fail. Any second thoughts? cc @tiancaiamao

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**